### PR TITLE
Updated documentation for creating a binding skeleton

### LINKED ISSUE #482: Skeleton readme outdated?
Referring openhab/openhab2-addons#2552

@maintainers Please assign me to this one.

> It appears the doc/readme on generating an OH2 skeleton is outdated.
> 
> It currently states claims only a single parameter is needed, when 2 are actually required:
> 
> For this, go into your git repository under git/openhab2-addons/addons/binding and call the script create_openhab_binding_skeleton.sh with a single parameter, which is your binding name in camel case (e.g. ‘ACMEProduct’ or ‘SomeSystem’)
> 
> Usage: create_openhab_binding_skeleton BindingIdInCamelCase BindingIdInLowerCase

### PR DESCRIPTION
**Pull request by cweitkamp**
- All scripts need tow arguments `BindingIdInCamelCase` and `Author`

Needs openhab/openhab2-addons#3157 to be merged.

Closes #482 

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

**Comment by Confectrician** [comment]
Hi @cweitkamp,

The dependency has been merged.
Anything to update due to sour last commits there or are we good to go with merging this?

File: developers/development/bindings.md [hunk 1]
```diff
@@ -22,9 +22,16 @@ As a first step, you need to decide in which namespace you want to develop your
...
-
-For the Eclipse SmartHome namespace: Choose the option "Eclipse SmartHome Extensions" in [your IDE setup](ide.html), and go ahead and create a skeleton for your binding. For this, go to `git/smarthome/tools/archetype`and run `mvn install` in order to install the archetype definition in your local Maven repo. Now go to `git/smarthome/extensions/binding` and call the script `create_esh_binding_skeleton.sh` with a single parameter, which is your binding name in camel case (e.g. 'ACMEProduct' or 'SomeSystem'). When prompted, enter your name as author and hit "Y" to start the skeleton generation.
+For the openHAB namespace: Choose the option "openHAB 2 Add-ons" in [your IDE setup](ide.html), and go ahead and create a skeleton for your binding.
+For this, go into your git repository under `git/openhab2-addons/addons/binding` and call the script `create_openhab_binding_skeleton.sh` with two arguments
```
Comment (wborn) [inline developers/development/bindings.md]: A period should be added at the end of this sentence.

**Comment by cweitkamp** [comment]
FTR: We currently we have to perform the following steps to create a new OH2 binding before we get it running. Shall we mention that in any way? Wdyt?

### New Binding:
- run `create_openhab_binding_skeleton.sh`
- replace `NOTICE` with `about.html` in `build-properties`
- run `mvn license:format`
- _minor_: apply formatter in "MyTestBindingConstants.java"

### New Binding Test:
- run `create_openhab_binding_test_skeleton.sh`
- replace `NOTICE` with `about.html` in `build-properties`
- remove import `org.hamcrest.collections` from `MANIFEST.md`
- run `mvn license:format`
- add the following snippet to `pom.xml`
```
            <dependency>
              <type>eclipse-plugin</type>
              <artifactId>org.eclipse.equinox.ds</artifactId>
              <version>0.0.0</version>
            </dependency>
```

**Comment by Confectrician** [comment]
>Shall we mention that in any way? Wdyt?

I would say yes.

**Comment by wborn** [comment]
Thanks @cweitkamp! That should make it more user friendly. Especially for new developers. :-)

**Comment by Confectrician** [comment]
Looks good @cweitkamp 

Shall we wait for eclipse/smarthome#5380 to be merged?
I would subscribe then to that PR and merge afterwards here.

**Comment by cweitkamp** [comment]
@Confectrician Thanks for your review.

In general I would say yes. But if we do so we have to wait until the next snapshot build of ESH because the changes will only become available after that.

**Comment by Confectrician** [comment]
> But if we do so we have to wait until the next snapshot build of ESH because the changes will only become available after that.

I would say its fine after the pr has been merged. 
The versioned docs are on snapshot level anyway,
so we never will have the exact state of alle repos covered here,
until we commit a new release related docs version.

When the PR got merged this is a good signal for "docs will be correct in the nearer future for this point".
For me that's enough.

**Comment by Confectrician** [comment]
I was just waiting some time, to be a bit closer to a next esh build. 😄 

@cweitkamp do you want to take care of the last review or should we merge it for now?

**Pull request merged**